### ISSUE TITLE
[Twitch Stream Panel does not show in Channel List of Discord Server set in Settings

### ISSUE THREAD

**Issue by DevilHunter1337** [Orrielel/BetterDiscordAddons#12]
It worked a little awhile ago, but not now. It still tells me when streams have gone online or any that are online when I select the Server I selected in the Settings.

**Comment by DevilHunter1337** [comment]
I tired joining the "Support" server but I don't have permission to speak, don't see anyone else talking, and I somehow don't even get to stay in it if I refresh Discord so... 

The button still works, tell me that people are indeed streaming with the viewer count, settings still work.. but that's it. I really want this to work again...

**Comment by DevilHunter1337** [comment]
So now this happens, and the plugin refuses to start. I really like this plugin, and wished it was still working... 

https://www.dropbox.com/s/3podx23aeul1hr4/TwitchPanelError.png?dl=0

**Comment by Orrielel** [comment]
Yeah, I've been neglecting it for a long time now. Can't promise that I'll update it, but I'll see if I can spare some time on it.
